promise.js

```diff
@@ -0,0 +1,64 @@
+var fs = require('fs');
+
+function step1(){
```

Comment:
@psinha20 Please function & var naming convention properly. What step1 signify here? use a specific variable instead.

Reply:
@psinha20 Always use one space after the fun step1().